Binary files wxorig/.DS_Store and wxnew/.DS_Store differ
Binary files wxorig/src/common/.DS_Store and wxnew/src/common/.DS_Store differ
diff -uNr wxorig/src/common/prntbase.cpp wxnew/src/common/prntbase.cpp
--- wxorig/src/common/prntbase.cpp	2011-01-03 20:30:30.000000000 -0500
+++ wxnew/src/common/prntbase.cpp	2011-03-01 17:15:34.000000000 -0500
@@ -381,6 +381,15 @@
 
 bool wxPrinter::Print(wxWindow *parent, wxPrintout *printout, bool prompt)
 {
+    if( !prompt && m_printDialogData.GetToPage() == 0 ) 
+ 	{
+        // no prompt, the program would not print anything
+        // because ToPage was initialized to zero
+        int minPage, maxPage, selFrom, selTo;
+        printout->GetPageInfo(&minPage, &maxPage, &selFrom, &selTo);
+        m_pimpl->GetPrintDialogData().SetFromPage( minPage );
+        m_pimpl->GetPrintDialogData().SetToPage( maxPage );
+ 	}
     return m_pimpl->Print( parent, printout, prompt );
 }
 
